onnx/numpy_helper.py

```diff
@@ -27,9 +27,9 @@ def bfloat16_to_float32(
     shift = lambda x: x << 16  # noqa: E731
```

Comment:
L9: Do you know why mypy in lintrunner is complaining here (missing attribute e.g., `MYPY/[attr-defined]: Module "onnx" has no attribute "AttributeProto"`)? Does it mean that it needs to build onnx from source as well, just like before?

Reply:
Just fixed by importing explicitly. It seems to be mypy was not able to follow the import * imports. Not sure what happened previously but this has been a source of mypy errors we haven seen both in this repo and in projects that uses onnx